wordpress-mobile/WordPress-Android
# Handles authorisation in image urls with redirection

Depends on https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/3018

## Description
This PR fixes an issue were images with http redirects do not load due to missing authorisation headers

Internal ref p1716409760977969-slack-C04TBSWP09M

⚠️ Please merge after https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/3018 is merged and the FluxC version on this PR is updated to point to trunk.

-----

## To Test:
1. Open the reader and head to a private blog (e.g. a P2)
2. Open a post with images in the comments
3. **Verify** that images with redirects load as expected

-----

## Regression Notes

1. Potential unintended areas of impact

    - Image loading

4. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual testing

5. What automated tests I added (or what prevented me from doing so)

    - Added test on the FluxC side

-----

## PR Submission Checklist:


-----

## Testing Checklist (strike-out the not-applying and unnecessary ones):
